### PR TITLE
Pizzeria debugged and fixed

### PR DESCRIPTION
**Pull request by NikczemnyOkularnik**
@NikczemnyOkularnik  @Crazyayane07  @Kasia9311

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: debuggers
🏅 Granted 10 XP for @NikczemnyOkularnik

**Comment by ziobron** [comment]
🏅 +1 XP for cooperation goes to @NikczemnyOkularnik

**Comment by ziobron** [comment]
✅ CI Passed - Task: debuggers
🏅 Granted 10 XP for @Crazyayane07

**Comment by ziobron** [comment]
🏅 +1 XP for cooperation goes to @Crazyayane07

**Comment by ziobron** [comment]
✅ CI Passed - Task: debuggers
🏅 Granted 10 XP for @Kasia9311

**Comment by ziobron** [comment]
🏅 +1 XP for cooperation goes to @Kasia9311

File: src/Pizzeria.cpp [hunk 1]
```diff
@@ -53,6 +54,7 @@ void Pizzeria::bakePizzas(int orderId)
         }
         std::get<Status>(*order) = Status::Baked;
     }
+    return;
```
Comment (michalsoloducha) [inline src/Pizzeria.cpp:57]: In this case the exception will never be thrown, no?

**Review by michalsoloducha — COMMENTED** [review]
See my comment for Pizzeria.cpp

**Comment by ziobron** [comment]
Thanks @michalsoloducha for Code Review.
🏅 1 XP granted

**Pull request closed without merging**